### ISSUE TITLE
Changing the Value property of an Icon using PseudoClasses does not reflect in the UI

### ISSUE THREAD

**Issue by JongHoon023** [Projektanker/Icons.Avalonia#138]
Hello. I am encountering an issue where the icon does not change when I modify the Value property of an Icon using PseudoClasses.

https://github.com/Projektanker/Icons.Avalonia/assets/38038052/9cc60705-2d46-48e1-8f45-5a3c8dd5416b

As shown in the video above, I changed the PseudoClasses every 3 seconds, but the icon does not change. The result I am looking for is as follows:

https://github.com/Projektanker/Icons.Avalonia/assets/38038052/ab29e2f7-7ca6-409d-a789-6d8e9d0346cc

Here is the code I have written:
```.axaml
<ResourceDictionary
	xmlns="https://github.com/avaloniaui"
    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
	xmlns:i="clr-namespace:Projektanker.Icons.Avalonia;assembly=Projektanker.Icons.Avalonia"
	xmlns:components="clr-namespace:Avalonia.Icon.PseudoClasses.Components;assembly=Avalonia.Icon.PseudoClasses">
	<ControlTheme
		x:Key="{x:Type components:IconTestView}"
		TargetType="components:IconTestView">
		<Setter
			Property="Template">
			<ControlTemplate>
				<Grid
					ColumnDefinitions="14, *"
					HorizontalAlignment="{TemplateBinding HorizontalAlignment}">
					<i:Icon
						x:Name="PART_Icon"
						Grid.Column="0"
						FontSize="24"
						RenderOptions.BitmapInterpolationMode="HighQuality" />

					<TextBlock
						x:Name="PART_StatusText"
						Grid.Column="1"
						Margin="4, 0, 8, 0"
						FontSize="30"
						Foreground="Violet" />
				</Grid>
			</ControlTemplate>
		</Setter>

		<!-- Icon -->
		<Style
			Selector="^:true /template/ i|Icon#PART_Icon">
			<Setter
				Property="Foreground"
				Value="#008926" />
			<Setter
				Property="Value"
				Value="fa-solid fa-circle-check" />
		</Style>

		<Style
			Selector="^:false /template/ i|Icon#PART_Icon">
			<Setter
				Property="Foreground"
				Value="#DD2929" />
			<Setter
				Property="Value"
				Value="fa-solid fa-circle-xmark" />
		</Style>

		<!-- TextBlock -->
		<Style
			Selector="^:true /template/ TextBlock#PART_StatusText">
			<Setter
				Property="Text"
				Value="IsTrue is true." />
		</Style>

		<Style
			Selector="^:false /template/ TextBlock#PART_StatusText">
			<Setter
				Property="Text"
				Value="IsTrue is false." />
		</Style>
	</ControlTheme>
</ResourceDictionary>
```

While creating a project that can reproduce this issue, I found one solution, which is that moving the window to a different monitor resolves this problem. 

You can refer to the link below for a project that can reproduce this issue.
https://github.com/JongHoon023/Avalonia.Icon.PseudoClasses

Thank you for creating such a wonderful project.

**Comment by just-seba** [comment]
Thank you very much for the best reproduction repository I've ever seen.
I looked into the problem and I think I found a solution. The issue should be fixed with [9.3.0](https://github.com/Projektanker/Icons.Avalonia/releases/tag/v9.3.0)

**Comment by JongHoon023** [comment]
Thank you very much for the quick fix! After checking version 9.3.0, I have confirmed that the issue has been resolved. :)
